docs.html

```diff
@@ -12,6 +12,8 @@
     <link rel="stylesheet" href="/Public/components/alert/alert.css">
     <link rel="stylesheet" href="/Public/components/avatar/avatar.css">
```

Comment:
In spite of adding all the css files in head tag, it should be imported in one style.css file & that style.css file should be put in head tag.